debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/RecordsSnapshotProducer.java

```diff
@@ -148,7 +151,7 @@ private void takeSnapshot(Consumer<SourceRecord> consumer) {
             // we're locking in SHARE UPDATE EXCLUSIVE MODE to avoid concurrent schema changes while we're taking the snapshot
             // this does not prevent writes to the table, but prevents changes to the table's schema....
             schema.tables().forEach(tableId -> statements.append("LOCK TABLE ")
-                                                         .append(tableId.toString())
+                                                         .append(tableId.toQuotedId(TableIdTransformer.DOUBLE_QUOTED))
```

Comment:
I think I've asked it before, but it still seems unaddressed: it seems we'll always quote the table name that way, correct? Shouldn't the transformer be instead be a part (member) of the table id which then would internally decide whether to apply it or not.